setup/user_transliteration.py

```
@@ -113,16 +112,18 @@ def read_hunspell_dict(self) -> str:
                 encoding = match.group('encoding')
                 print("load_dictionary(): encoding=%(enc)s found in %(aff)s" %{
                     'enc': encoding, 'aff': self.aff_file})
+        if not encoding:
+            encoding = 'UTF-8'
         try:
-            dict_buffer = codecs.open(
-                self.hunspell_dict, encoding=encoding).read().replace('\r\n', '\n')
+            with open(self.hunspell_dict, encoding=encoding) as f:
+                dict_buffer = f.read().replace('\r\n', '\n')
         except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)